ceph-windows-pull-requests/config/definitions/ceph-windows-pull-requests.yml

```diff
@@ -0,0 +1,89 @@
+- job:
+    name: ceph-windows-pull-requests
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: amd64 && focal && libvirt
+    display-name: 'ceph-windows: Pull Requests'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ionutbalutoiu/ceph/
+      - rebuild:
+          auto-rebuild: true
+      - inject:
+          properties-content: |
+            TERM=xterm
+
+    parameters:
+      - string:
+          name: ghprbPullId
+          description: "The GitHub pull request id, like '72' in 'ceph/pull/72'"
+
+    triggers:
+      - github-pull-request:
+          cancel-builds-on-update: true
+          allow-whitelist-orgs-as-admins: true
+          org-list:
+            - ceph
+          trigger-phrase: 'jenkins test windows'
+          skip-build-phrase: '^jenkins do not test.*'
+          only-trigger-phrase: false
+          github-hooks: true
+          permit-all: true
+          auto-close-on-fail: false
+          status-context: "ceph windows tests"
+          started-status: "running ceph windows tests"
+          success-status: "ceph windows tests succeeded"
+          failure-status: "ceph windows tests failed"
+
+    scm:
+      - git:
+          url: https://github.com/ionutbalutoiu/ceph.git
```

Comment:
Is this just for testing purposes?

Reply:
Yes, correct. I fixed this in the last PR update.